packages/cms/src/components/Viz/Viz.jsx

```diff
@@ -20,6 +20,10 @@ class Viz extends Component {
     if (updateSource && resp.source) updateSource(resp.source);
   }
 
+  setVariables(newVariables) {
+    if (this.props.onSetVariables) this.props.onSetVariables(newVariables);
```

Comment:
Why do we have this pass-through function (don't all `Viz` components have access to `onSetVariables`?), and why aren't we using `context` here like we do for `updateSource`?